### PR TITLE
Feature/loginscreen

### PR DESCRIPTION
**Pull request by JuliaSchw**

### CODE REVIEW

File: pages/api/auth/[...nextauth].js [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+              image:
+                "https://avatars.githubusercontent.com/u/41701911?s=200&v=4",
+            };
+          },
+        })
+      : GithubProvider({
+          clientId: process.env.GITHUB_ID,
+          clientSecret: process.env.GITHUB_SECRET,
+        }),
+  ],
```
Comment (F-Kirchhoff) [inline pages/api/auth/[...nextauth].js:32]: please make the github signin in the preview mode working on vercel. You can use the "branch" link to your application then you don't need to update the callback link with every commit. If you cant find it let me know.

File: pages/api/projects/index.js [hunk 2]
```diff
@@ -1,17 +1,20 @@
...
   if (request.method === "GET") {
     const projects = await Project.find();
     return response.status(200).json(projects);
   }
-  if (request.method === "POST") {
+  if (session && request.method === "POST") {
     try {
       const newProject = request.body;
-      await Project.create(newProject);
+      await Project.create({ ...newProject, author: session.user.name });
```
Comment (F-Kirchhoff) [inline pages/api/projects/index.js:17]: nice

**Review by F-Kirchhoff — CHANGES_REQUESTED** [review]
Hey!
looking good so far, please make the github auth work in your preview and then remove the ternary operator to switch to the credentials provider if you are in the preview mode.

**Review by On-Fi — COMMENTED** [review]
LGTM 👍🏼

**Pull request merged**